install/helm/open-match/values.yaml

```diff
@@ -106,11 +106,13 @@ configs:
 redis:
   fullnameOverride: om-redis
   redisPort: 6379
-  usePassword: true
-  usePasswordFile: true
+  usePassword: false
```

Comment:
Reason for this?

Reply:
This is a minor twist to unify the security settings. It feels a bit off by having the TLS settings for Open Match core disabled and Redis TLS enabled.

Do you mind if I have it checked in along with this fix?